modules/jax/package.json

```
@@ -11,8 +11,8 @@
     "prepublishOnly": "yarn run lint && yarn run build",
     "lint": "eslint --config ../../.eslintrc.json src/**/*.ts",
     "build": "tsc -b",
-    "test": "bedrock-auto -b chrome-headless --customRoutes src/test/json/routes.json -d src/test/ts/",
-    "test-manual": "bedrock --customRoutes src/test/json/routes.json -d src/test/ts/",
+    "test": "bedrock-auto -b chrome-headless --customRoutes ../tinymce/src/core/test/json/routes.json -d src/test/ts/",
+    "test-manual": "bedrock --customRoutes ../tinymce/src/json/routes.json -d src/test/ts/",
```

I assume they are meant to be the same.

```suggestion
    "test-manual": "bedrock --customRoutes ../tinymce/src/core/test/json/routes.json -d src/test/ts/",
```

---

Good catch changed this.